account_bank_statement_import_auto_reconcile/models/account_bank_statement_import_auto_reconcile_rule.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please add triple quote __doc__ strings to some of the functions in this module to add some documentation, as a lot of trickery is going on, the purpose and workings of which can a little bit be guessed, but explanation would be welcome.